licenses/licenses.json

```diff
@@ -1,10 +1,2751 @@
 {
-    "EasyBuild": {
-        "spdx": "GPL-2.0-only",
-        "license_url": "https://easybuild.io"
+    "ALL/0.9.2-foss-2023a": {
+        "License": "not found",
+        "Needs Manual Check": true,
+        "Retrieved From": "not found"
     },
-    "GCCcore": {
-        "spdx": "GPL-2.0-with-GCC-exception",
-        "license_url": "https://github.com/gcc-mirror/gcc/blob/master/COPYING"
+    "AOFlagger/3.4.0-foss-2023b": {
+        "License": "not found",
+        "Needs Manual Check": true,
+        "Retrieved From": "not found"
+    },
+    "ASE/3.22.1-gfbf-2022b": {
+        "License": "not found",
+        "Needs Manual Check": true,
+        "Retrieved From": "not found"
+    },
+    "ATK/2.38.0-GCCcore-13.2.0": {
+        "License": [
+            "Other"
+        ],
+        "Needs Manual Check": false,
+        "Retrieved From": "https://developer.gnome.org/atk/"
+    },
+    "Abseil/20240116.1-GCCcore-13.2.0": {
+        "License": [
+            "Apache-2.0"
+        ],
+        "Needs Manual Check": false,
+        "Retrieved From": "https://abseil.io/"
+    },
+    "Archive-Zip/1.68-GCCcore-12.2.0": {
+        "License": [
+            "Other"
+        ],
+        "Needs Manual Check": false,
+        "Retrieved From": "https://metacpan.org/pod/Archive::Zip"
+    },
+    "Armadillo/12.8.0-foss-2023b": {
+        "License": [
+            "Apache-2.0"
+        ],
+        "Needs Manual Check": false,
+        "Retrieved From": "https://arma.sourceforge.net/"
+    },
+    "Arrow/16.1.0-gfbf-2023b": {
+        "License": [
+            "Other"
```

Comment:
License for this is https://github.com/apache/arrow/blob/main/LICENSE.txt ("Apache-2.0")